src/components/split-button/split-button-children.style.ts

```diff
@@ -9,13 +9,34 @@ type StyledSplitButtonChildrenContainerProps = {
 };
 
 const StyledSplitButtonChildrenContainer = styled.div<StyledSplitButtonChildrenContainerProps>`
+  border-radius: var(--borderRadius100);
   ${({ theme, align, minWidth }) => css`
     background-color: var(--colorsActionMajorYang100);
     min-width: ${minWidth}px;
     white-space: nowrap;
     z-index: ${theme.zIndex.popover};
     box-shadow: var(--boxShadow100);
 
+    > {
```

Comment:
This will have the same issue as mentioned before, copy the approach I've suggested in `MultiActionButton`